WebDevStudios/wd_s
# Feature/#739 preload styles scripts

Closes #739

> I branched off #728 (since that will be the new `main` branch). Therefore, this PR requires #728 be merged first!

### DESCRIPTION

- Created a function that will add theme style/script preloading to the `<head>`

### SCREENSHOTS

![screenshot](https://dl.dropbox.com/s/ayj96s3yygox3gw/Screen%20Shot%202021-09-16%20at%207.36.19%20AM.png?dl=0)

![screenshot](https://dl.dropbox.com/s/khibac1t7dnkx9r/Screen%20Shot%202021-09-16%20at%207.37.41%20AM.png?dl=0)

### STEPS TO VERIFY

1. `gh pr checkout 740`
2. View your local
3. Open Chrome, Firefox, or Safari dev tools and look at `<head>`
4. Verify the theme style/script are `<link rel="preload" ...>`
